source/index.md

```diff
@@ -539,9 +539,10 @@ payment method must be omitted.
   </dd>
 </dl>
 
-##### Method: `card`
+---
+##### **Method**: `card`
 
-<dl class="dl-horizontal">
+<dl>
```

Comment:
The width is defined by `bootstrap` and the conversion of the document, we can change it, but it might be hacky.
Besides, we want a limited page width. The problem I think is the formatting of the `<dd>`. The wrapping is done horribly.
Let's just public LaTeX docs! :)